android/app/src/main/res/values-es/strings.xml

```diff
@@ -1301,37 +1301,37 @@
 	<!-- These translations are used for all type.highway.*.tunnel. -->
 	<string name="type.highway.track.tunnel">Túnel</string>
 	<string name="type.highway.traffic_signals">Semáforos</string>
-	<string name="type.highway.trunk">Calle</string>
+	<string name="type.highway.trunk">Vía rápida</string>
```

Comment:
I would prefer "Via troncal". There are lots of trunk with low maxspeed...

Reply:
Changed to "Vía troncal".

Please check [PR 7558](https://github.com/organicmaps/organicmaps/pull/7558).